src/function/arithmetic/multiply.js

```diff
@@ -127,7 +127,7 @@ export const createMultiply = /* #__PURE__ */ factory(name, dependencies, ({ typ
     let mf = multiplyScalar
 
     // process data types
-    if (adt && bdt && adt === bdt && typeof adt === 'string') {
+    if (adt && bdt && adt === bdt && typeof adt === 'string' && adt !== 'mixed') {
```

Comment:
When I remove these `&& adt !== 'mixed'` cases, there is no difference in performance, and all unit tests still pass. Can you add a unit test if you added this to solve specific errors?

Reply:
Nevermind, I see that there are actually failing tests when removing the conditions that check for `'mixed'`